### PR TITLE
Update tic_tac_toe.py

### PR DESCRIPTION
**Pull request by harshitashyale**
Made some minor fixes

### CODE REVIEW

**Comment by githubparticipant** [comment]
Seems like there's no score updating logic?

**Comment by harshitashyale** [comment]
That's for next release

File: tic_tac_toe.py [hunk 1]
```diff
@@ -19,14 +19,14 @@ def printBoard(playerY, playerX):
     print(f"| {six} | {seven} | {eight} |") 
 
 
-def checkwinner(playerX, playerY):
+def checkwinner(playerY, playerX):
```
Comment (githubparticipant) [inline tic_tac_toe.py:22]: switch the variables

Comment (harshitashyale) [reply tic_tac_toe.py:22]: do you mean (playerX, playerY)?

Comment (githubparticipant) [reply tic_tac_toe.py:22]: make the code simpler

Comment (githubparticipant) [reply tic_tac_toe.py:22]: yes

File: tic_tac_toe.py [hunk 2]
```diff
@@ -19,14 +19,14 @@ def printBoard(playerY, playerX):
...
     wins = [[0, 1, 2], [3, 4, 5], [6, 7, 8],[0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]]
     for win in wins:
         if(sum(playerX[win[0]], playerX[win[1]], playerX[win[2]]) == 3):
-            print("Player Y has won the game")
```
Comment (githubparticipant) [inline tic_tac_toe.py:26]: recheck logic

File: tic_tac_toe.py [hunk 3]
```diff
@@ -19,14 +19,14 @@ def printBoard(playerY, playerX):
...
+            print("Player X has won the game")
             return 1
         if(sum(playerY[win[0]], playerY[win[1]], playerY[win[2]]) == 3):
-            print("Player Y has won the game")
+            print("Player X has won the game")
```
Comment (githubparticipant) [inline tic_tac_toe.py:29]: recheck

File: tic_tac_toe.py [hunk 4]
```diff
@@ -19,14 +19,14 @@ def printBoard(playerY, playerX):
...
 
-def checkwinner(playerX, playerY):
+def checkwinner(playerY, playerX):
     wins = [[0, 1, 2], [3, 4, 5], [6, 7, 8],[0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]]
     for win in wins:
         if(sum(playerX[win[0]], playerX[win[1]], playerX[win[2]]) == 3):
-            print("Player Y has won the game")
+            print("Player X has won the game")
             return 1
         if(sum(playerY[win[0]], playerY[win[1]], playerY[win[2]]) == 3):
```
Comment (githubparticipant) [inline tic_tac_toe.py:28]:
```suggestion
        if(sum(playerY[win[0]], playerY[win[1]], playerY[win[2]]) == 3):
```

File: tic_tac_toe.py [hunk 5]
```diff
@@ -19,14 +19,14 @@ def printBoard(playerY, playerX):
     print(f"| {six} | {seven} | {eight} |") 
 
 
-def checkwinner(playerX, playerY):
+def checkwinner(playerY, playerX):
     wins = [[0, 1, 2], [3, 4, 5], [6, 7, 8],[0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]]
     for win in wins:
         if(sum(playerX[win[0]], playerX[win[1]], playerX[win[2]]) == 3):
-            print("Player Y has won the game")
+            print("Player X has won the game")
```
Comment (githubparticipant) [inline tic_tac_toe.py:26]: recheck logic

File: tic_tac_toe.py [hunk 6]
```diff
@@ -19,14 +19,14 @@ def printBoard(playerY, playerX):
...
             return 1
         if(sum(playerY[win[0]], playerY[win[1]], playerY[win[2]]) == 3):
-            print("Player Y has won the game")
```
Comment (githubparticipant) [inline tic_tac_toe.py:29]: recheck

File: tic_tac_toe.py [hunk 7]
```diff
@@ -43,7 +43,7 @@ def checkwinner(playerX, playerY):
             value = int(input("Enter your Value: "))
             playerX[value] = 1
         else:
-            print("O's Chance to Play")
+            print("Y's Chance to Play")
             value = int(input("Enter your Value: "))
             playerY[value] = 1
```
Comment (githubparticipant) [inline tic_tac_toe.py:48]:
```suggestion
            playerY[value] = 0
```

**Comment by githubparticipant** [comment]
Make simplier!

File: tic_tac_toe.py (hunk 1 above)
Comment (githubparticipant) [reply tic_tac_toe.py:22]: Yes!

File: tic_tac_toe.py (hunk 5 above)
Comment (githubparticipant) [inline tic_tac_toe.py:26]: Recheck this logic!

File: tic_tac_toe.py (hunk 7 above)
Comment (githubparticipant) [inline tic_tac_toe.py:48]: Make this 0

File: tic_tac_toe.py [hunk 8]
```diff
@@ -43,7 +43,7 @@ def checkwinner(playerX, playerY):
             value = int(input("Enter your Value: "))
             playerX[value] = 1
         else:
-            print("O's Chance to Play")
```
Comment (githubparticipant) [inline tic_tac_toe.py:46]: recheck this pls

**Pull request closed without merging**